### PR TITLE
Renames and clean up

### PR DESCRIPTION
**Pull request by roysc**
Fix some lingering renames and remove old Makefile stuff

### CODE REVIEW

File: docker-compose.test.yml [hunk 1]
```diff
@@ -5,12 +5,12 @@ services:
     restart: on-failure
     depends_on:
       - ipld-eth-db
-    image: cerc-io/ipld-eth-db
```
Comment (i-norden) [inline docker-compose.test.yml:8]: This must have not worked prior to this fix?

**Review by i-norden — APPROVED** [review]
LGTM!

File: docker-compose.test.yml (hunk 1 above)
Comment (i-norden) [reply docker-compose.test.yml:8]: Hmm wait I'm second guessing this change. Shouldn't it be cerc-io?

Comment (i-norden) [reply docker-compose.test.yml:8]: Aka https://github.com/cerc-io/ipld-eth-server/blob/v4/docker-compose.yml#L8

Comment (i-norden) [reply docker-compose.test.yml:8]: But I also see https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/compose/docker-compose-ipld-eth-db.yml#L9 being used

In the publish actions it seems to be cerc-io though: https://github.com/cerc-io/ipld-eth-db/blob/v4/.github/workflows/publish.yaml#L19

Comment (roysc) [reply docker-compose.test.yml:8]: Yeah, stack-orchestrator uses `cerc/` for all Docker tags and `cerc-io` for Git repos. I figured `cerc/` would be the current prefix for local stack integration. It seems `git.vdb.to/cerc-io` is used for all CI actions, maybe @ABastionOfSanity can confirm.

In any case, I should make this consistent in the other compose file.

Comment (i-norden) [reply docker-compose.test.yml:8]: Yeah we should try to keep it consistent

**Pull request merged**